src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPackageInstaller.cs

```diff
@@ -541,7 +541,7 @@ private SourceRepository GetSource(string source)
 
             using (var sourceCacheContext = new SourceCacheContext())
             {
-                ResolutionContext resolution = new ResolutionContext(
+                var resolution = new ResolutionContext(
```

Comment:
Personally I would revert this `var` change and the `string` casing change not to reduce risk but to simply drop this PR to just the one line change and help folks focus on the real issue and fix.